app/backend/src/couchers/models.py

```diff
@@ -74,50 +93,68 @@ class User(Base):
     geom_radius = Column(Float, nullable=True)
     # the display address (text) shown on their profile
     city = Column(String, nullable=False)
+    hometown = Column(String, nullable=True)
 
     joined = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
     last_active = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
 
     # display name
     name = Column(String, nullable=False)
     gender = Column(String, nullable=False)
+    pronouns = Column(String, nullable=True)
     birthdate = Column(Date, nullable=False)  # in the timezone of birthplace
 
     # name as on official docs for verification, etc. not needed until verification
     full_name = Column(String, nullable=True)
 
     hosting_status = Column(Enum(HostingStatus), nullable=True)
+    meetup_status = Column(Enum(MeetupStatus), nullable=True)
 
     # verification score
     verification = Column(Float, nullable=True)
     # community standing score
     community_standing = Column(Float, nullable=True)
 
     occupation = Column(String, nullable=True)
+    education = Column(String, nullable=True)
     about_me = Column(String, nullable=True)
+    my_travels = Column(String, nullable=True)
+    things_i_like = Column(String, nullable=True)
     about_place = Column(String, nullable=True)
-    # profile color
-    color = Column(String, nullable=False, default="#643073")
     avatar_filename = Column(String, nullable=True)
     # TODO: array types once we go postgres
     languages = Column(String, nullable=True)
     countries_visited = Column(String, nullable=True)
     countries_lived = Column(String, nullable=True)
+    additional_information = Column(String, nullable=True)
 
     is_banned = Column(Boolean, nullable=False, default=False)
 
     # hosting preferences
     max_guests = Column(Integer, nullable=True)
-    multiple_groups = Column(Boolean, nullable=True)
     last_minute = Column(Boolean, nullable=True)
+    has_pets = Column(Boolean, nullable=True)
     accepts_pets = Column(Boolean, nullable=True)
+    pet_details = Column(String, nullable=True)
+    has_kids = Column(Boolean, nullable=True)
     accepts_kids = Column(Boolean, nullable=True)
+    kid_details = Column(String, nullable=True)
+    has_housemates = Column(Boolean, nullable=True)
+    housemates_details = Column(String, nullable=True)
```

Comment:
```suggestion
    housemate_details = Column(String, nullable=True)
```
This matches what is in the rest of the code

Reply:
This is still present